config/locales/en.yml

```diff
@@ -121,3 +121,28 @@ en:
         roles_and_permissions:        'Users and groups'
         system_status:                'System status'
         technical:                    'Technical'
+  proprietor:
```

Comment:
Can you run this on Hyku once all the translations have been added: https://github.com/samvera-labs/hyrax/commit/dfe0c350bb1f5db9a6c01e186ecb852b5ab8c994#diff-54441a884831d33711a0af9d92a20a32

This lets us keep our locale files looking similar.